docs/sources/datasources/tempo/_index.md

```diff
@@ -46,7 +46,7 @@ Set the data source's basic configuration options carefully:
 
 You can also configure settings specific to the Tempo data source:
```

Comment:
Changing this to a period since this sentence isn't followed by a bulleted or numbered list. We could add a bulleted item for each of the subheadings that explain how to configure settings (there are a lot) or just add a note that the options are described below. For example: 
```suggestion
You can also configure settings specific to the Tempo data source. These options are described in the sections below.   
```